src/fsharp/symbols/Symbols.fsi

```diff
@@ -900,13 +909,19 @@ type FSharpMemberOrFunctionOrValue =
     
     /// Indicated if this is a value
     member IsValue: bool
+    
+    /// Indicated if this is a value
+    member IsFunction : bool
 
     /// Indicates if this is a constructor.
     member IsConstructor: bool
     
     /// Format the type using the rules of the given display context
-    member FormatLayout: context: FSharpDisplayContext -> Layout
-
+    member FormatLayout: displayContext: FSharpDisplayContext -> Layout
+    
+    /// Format the type using the rules of the given display context
```

Comment:
How does this member differ from `FormatLayout`? Is it possible to add an example to the comment here, or clarify what `the type` is in this context?

Reply:
`FormatLayout` gives the full type siganture. So if a function is `int -> int`, that's what it would print. I wanted only the return type signature.

Reply:
In general I think these kinds of things - "get me the format of this thing, but <this way>" is kinda lame to shove on MemberOrFunctionOrValue, but...eh, it's what we got right now